x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/add_prepackaged_rules_route.ts

```diff
@@ -117,24 +115,12 @@ export const createPrepackagedRules = async (
   );
   const prepackagedRules = await getExistingPrepackagedRules({
     rulesClient,
-    isRuleRegistryEnabled: ruleRegistryEnabled,
   });
   const rulesToInstall = getRulesToInstall(latestPrepackagedRules, prepackagedRules);
   const rulesToUpdate = getRulesToUpdate(latestPrepackagedRules, prepackagedRules);
   const signalsIndex = siemClient.getSignalsIndex();
-  if (!ruleRegistryEnabled && (rulesToInstall.length !== 0 || rulesToUpdate.length !== 0)) {
-    const signalsIndexExists = await getIndexExists(esClient.asCurrentUser, signalsIndex);
-    if (!signalsIndexExists) {
-      throw new PrepackagedRulesError(
-        `Pre-packaged rules cannot be installed until the signals index is created: ${signalsIndex}`,
-        400
-      );
```

Comment:
Is this error still possible even with the RuleRegistry? Like if you create a new space, but don't go to the security app, and then hit this API directly the indices won't have been created yet, no? RuleRegistry index creation happens on Kibana startup for existing spaces, but new spaces require an alert to be generated, is that correct?

Reply:
@spong I'll have to test this scenario, but you may be right. I think I'll follow up with an additional PR/issue to address this and the `output_index` stuff.